docs/dev/build_linux.md

```diff
@@ -65,7 +65,7 @@ You can use the following additional build options:
    cmake -DCMAKE_TOOLCHAIN_FILE=<openvino_repo>/cmake/toolchains/ia32.linux.toolchain.cmake ..
    ```
 
-- If you don't want to use the GPU plugin, use the `-DENABLE_INTEL_GPU=OFF` CMake build option and skip the installation of the Intel® Graphics Compute Runtime for OpenCL™ Driver.
+- OpenVINO offers several CMake options that can be used to build a custom OpenVINO runtime, which can speed up compilation. These options allow you to skip compilation of other devices and frontends by disabling/enabling them. You can find a list of available options [here](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/cmake_options_for_custom_compilation.md)
```

Comment:
`skip compilation of other devices` -> `skip compilation of other plugins`

Reply:
Change applied in https://github.com/openvinotoolkit/openvino/pull/18101/commits/f5d6f99bd00e794e0bb880bdc48933991f80bc56